lib/betterdoc/website/templating/dialect.rb

```diff
@@ -0,0 +1,9 @@
+module Betterdoc
```

Comment:
What purpose does this class serve?

Reply:
It started out as the base class for all dialects until it so happened that there actually wasn't any functionality to be put in the base class. It could be removed but my strong-typed-programming-language brain can't bring itself to that ;-)

Reply:
Hm, so we should remove it, it is needless clutter, right?